www/templates/components/main/_histogram-container.html

```diff
@@ -0,0 +1,8 @@
+<div>
+
+    {{$ctrl.data}}
+
+    <filter-container filter="$ctrl.filter" on-filter-change="$ctrl.toggle($event.filter)"></filter-container>
+    <histogram-chart data-dataset="$ctrl.data"></histogram-chart>
```

Comment:
Previously, you had `<histogram-chart data-set="$ctrl.data"></histogram-chart>` If you still wanted to use dataSet as data-set, you would have to write `data-data-set="$ctrl.data"`